Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/MediaPlayerElement/MediaPlayerElementDesigner.cs

```diff
@@ -20,7 +20,6 @@ public override void InitializeNewComponent(IDictionary defaultValues)
                 // Set MinimumSize in the designer, so that the control doesn't go to 0-height
                 control.MinimumSize = new System.Drawing.Size(100, 100);
                 control.Dock = System.Windows.Forms.DockStyle.Fill;
-                control.AutoPlay = false;
```

Comment:
Why this had to be removed?

Reply:
I used the default values for all properties from the UWP control and AutoPlay is true by default. I guess this line was added to avoid actually playing the video in the designer. This is not necessary anymore because the UWP control is no longer instantiated in the designer.